Code/Framework/AzFramework/AzFramework/Visibility/OcclusionBus.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Entity.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Math/Aabb.h>
+#include <AzCore/Math/Matrix4x4.h>
+#include <AzCore/Memory/SystemAllocator.h>
+#include <AzCore/Name/Name.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzFramework/Entity/EntityContextBus.h>
+
+namespace AZ
+{
+    class ReflectContext;
+}
+
+namespace AzFramework
+{
+    //! OcclusionRequests is an interface for creating occlusion views and queries. The bus must be connected to an entity context ID for an
+    //! entity context containing objects that contribute to or interact with the occlusion scene.
+    class OcclusionRequests : public AZ::EBusTraits
+    {
+    public:
+        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        typedef AzFramework::EntityContextId BusIdType;
+        using MutexType = AZStd::mutex;
```

Comment:
Suggestion: Consider a recursive_mutex here, I've had to change several EBus implementations over time from mutex to recursive_mutex to fix bugs. Specifically, all it takes is one EBus method that triggers a notification or calls out to another system, that in some scenario calls back into the Occlusion code for whatever reason. Maybe it's not happening here right now or doesn't seem likely, but I'm not sure there's much benefit in making this intentionally non-recursive.